song/song_server.py

```diff
@@ -122,3 +133,72 @@ def _send_init_to_display(self):
         self.display_client.send_message(settings.DISPLAY_INIT_ADDRESS,
                                          pickle.dumps(self.song_machine.parser.categories, protocol=2)
                                          )
+
+class Tonality:
+    '''
+    idea of tonality counter combined with rules for FX chains and Sample Triggering in
+    Ableton Live
+    '''
+
+    tonality_counter = Counter()
+    FX_KEY = 'Clean'
+    ctrl_val = 0
+    last_cats = []
+    last_value = 0
+    most_common = ''
+    tonality_lock = False
```

Comment:
1. Please move the variables that can change their value into the `__init__`-function
2. I think the prefix `tonality_` in the variables `tonality_counter` and `tonality_lock` can be removed, don't you?